src/pc/pl_arc.c

```diff
@@ -0,0 +1,40 @@
+#include <game.h>
+#include <log.h>
+#include "pc.h"
+
+u8* g_PlOvlSpritesheet[256];
+static u8* sprite_data = NULL;
+
+void InitPlayerArc(FileLoad* file) {
+    int i;
+
+    g_PlayableCharacter = PLAYER_ALUCARD;
+
+    if (sprite_data) {
+        free(sprite_data);
+    }
+    // this is not freed anywhere else, technically a memory leak but it's not
```

Comment:
You mean because it doesn't continue to grow at runtime?

Reply:
It is because it is not freed before the application exits. I've been meaning to implement a wrapper around `atexit` to handle these kind of scenarios without coupling the code too much.

It never grows though. If the init function is called twice, it just frees the previous one

Reply:
I think we should turn on leaksanitizer eventually but it's not a big deal right now https://clang.llvm.org/docs/LeakSanitizer.html

Reply:
Let's keep that in mind, it definitively sounds like a good option!